arangod/Pregel/GraphStore/GraphStore.cpp

```diff
@@ -370,6 +246,8 @@ void GraphStore<V, E>::loadVertices(
   uint64_t numVertices =
       coll->numberDocuments(&trx, transaction::CountType::Normal);
 
+  auto vertices = std::vector<Vertex<V, E>>(numVertices);
```

Comment:
I don't see where this variable is modified, you just return this emtpy vector from the function.